apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -48,13 +48,47 @@ export function updateDefaultList(listData) {
     });
 }
 
+export function uploadDefaultListMetadata(metadata, environment) {
+  return fetch(
+    `/api/v1/animation-library/default-spritelab-metadata/${environment}`,
+    {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json'
+      },
+      body: JSON.stringify(metadata)
+    }
+  )
+    .then(response => {
+      if (!response.ok) {
+        throw new Error(
+          `Default Sprite Metadata Upload Error(${response.status}: ${
+            response.statusText
+          })`
+        );
+      }
+      return Promise.resolve();
+    })
+    .catch(err => {
+      return Promise.reject(err);
+    });
+}
+
 // Returns the metadata of the list of default sprites in SpriteLab in English
 export function getDefaultListMetadata() {
   return fetch('/api/v1/animation-library/default-spritelab-metadata').then(
     response => response.json()
   );
 }
 
+export function moveDefaultSpriteMetadataToProduction() {
+  // Get metadata from levelbuilder
+  return getDefaultListMetadata().then(metadata => {
+    // Put metadata on production
+    return uploadDefaultListMetadata(metadata, 'production');
```

Comment:
what should we do if `getDefaultListMetadata` fails?

Reply:
Added catch branch.